services/offers_service.py

```diff
@@ -1,6 +1,10 @@
 from datetime import date, datetime
 from matplotlib import pyplot as plt, dates as mdates
 from helpers import programmes_helper
+import sys
+sys.path.insert(1,'/path/to/application/app/folder')
```

Comment:
Similarly to my previous comment, can you remove this part?

Reply:
Without this part, we won't be able to navigate out of the current folder, one easy way is making another file inside the same directory